React/Base/RCTConvert.m

```diff
@@ -614,11 +616,114 @@ + (type)type:(id)json                                       \
       @"systemPurpleColor": @{}, // 10_10
       @"systemRedColor": @{},    // 10_10
       @"systemYellowColor": @{}, // 10_10
+#else // TARGET_OS_IOS
+      // https://developer.apple.com/documentation/uikit/uicolor/ui_element_colors
+      // Label Colors
+      @"labelColor": @{ // 13
```

Comment:
Without the context of 10_10 above, it was very unclear what the "13" means. Probably doesn't hurt to precede with iOS13 or something like that instead